front-end/src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,165 @@
+        marginLeft: '50px',
+        '&:hover': {
+            backgroundColor: '#1261A3'
+        }
+    },
+    label: {
+        textTransform: 'capitalize',
+        fontSize: '14px',
+        fontWeight: '500'
+    },
+})(Button);
+
+export default function Navbar() {
+    const classes = useStyles();
+    const [state, setState] = useState({
+        type: '',
+    });
+    const handleChange = (event) => {
+        const name = event.target.name;
+        setState({
+            ...state,
+            [name]: event.target.value,
+        });
+    };
+
+    return (
+        <div className={classes.root}>
+            <Grid item container className={classes.navbar} alignItems="center">
+                <Grid item container sm alignItems="center" className={classes.logoWrap}>
+                    <Grid item className={classes.logo}>
+                        <img src={logo} className={classes.logoImage} />
+                    </Grid>
+                    <LogoTitle>Civic</LogoTitle>
+                </Grid>
+                <Grid item container sm alignItems="center" className={classes.searchWrap}>
+                    <Grid item>
+                        <FormControl className={classes.formControl}>
+                            <InputLabel htmlFor="type-select" className={classes.inputLabel}>Status</InputLabel>
+                            <Select
+                                native
+                                value={state.type}
+                                onChange={handleChange}
+                                inputProps={{
+                                    name: 'type',
+                                    id: 'type-select',
+                                }}
+                            >
+                                <option aria-label="" value="" />
+                                <option value="Proposed">Proposed</option>
+                                <option value="Reviewing by gov">Reviewing by gov</option>
+                                <option value="Approved by gov">Approved by gov</option>
+                                <option value="Rejected by gov">Rejected by gov</option>
+                                <option value="Voteable">Voteable</option>
+                                <option value=" Vote passed"> Vote passed</option>
+                                <option value="Vote failed">Vote failed</option>
+                                <option value="Executing by gov">Executing by gov</option>
+                                <option value="Closed">Closed</option>
+                            </Select>
+                        </FormControl>
+                    </Grid>
+                    <Grid item className={classes.searchIcon}>
+                        <SearchIcon />
+                    </Grid>
+                </Grid>
+                <Grid item container sm alignItems="center" className={classes.buttonWrap}>
+                    <NavbarTitle>Dashboard</NavbarTitle>
```

Comment:
"Dashboard" needs to be clickable like a button to navigate to the dashboard. Please use a button element that is styled so that it looks the same as now but is clickable.